lib/registration.pm

```diff
@@ -241,7 +241,15 @@ sub select_addons_in_textmode {
     if ($flag) {
         send_key_until_needlematch 'scc-module-area-selected', 'tab';
         send_key_until_needlematch "scc-module-$addon",        'down';
-        send_key 'spc';
+        if (check_var('ARCH', 'aarch64') && check_var('HDDVERSION', '12-SP2')) {
+            if (check_screen('scc-module-tcm-selected', 5)) {
```

Comment:
please "merge" this two ifs
if (check_var('ARCH', 'aarch64') && check_var('HDDVERSION', '12-SP2') && check_screen('scc-module-tcm-selected', 5)) {